x-pack/plugins/canvas/tasks/helpers/webpack.plugins.js

```diff
@@ -6,6 +6,9 @@
 
 const path = require('path');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
+const {
+  createServerCodeTransformer,
+} = require('@kbn/interpreter/tasks/build/server_code_transformer');
```

Comment:
The interpreter seems like a weird place to import build configuration from. It's also weird that it's importing something that wouldn't be part of the published package.

Do you think putting this in another package (currently existing or not) would make more sense?

Reply:
We really don't want to have hundreds of packages in two years, so no to another package. I think this is fine, we don't publish these packages and if we want to we can figure something out then.